src/main/arraytests.java

```diff
@@ -0,0 +1,28 @@
+package tests;
+
+import static org.junit.Assert.*;
+
+import java.lang.reflect.Array;
+
+import org.junit.Test;
+
+public class arraytests {
+
+	@Test
+	public void test() {
+		MedTask med = new MedTask();
+		int[] newArray = new int[3];

```

Comment:
On your arraytests JUnit test case, you are initializing newArray to length 3, but then setting
"newArray[3] = 40;" even though [3] would be the fourth index.